src/Components/LatestEpisodes/LatestEpisodes.js

```diff
@@ -1,25 +1,51 @@
-import React from "react";
+import React, { useState } from "react";
 import Episode from "../Episode/Episode";
+import { Link } from "react-router-dom";
+import ReactSimplyCarousel from "react-simply-carousel";
 import "./LatestEpisodes.css";
 
-export default function LatestEpisodes() {
+export default function LatestEpisodes(props) {
+    const { episodes } = props;
+
+    const [activeSlideIndex, setActiveSlideIndex] = useState(0);
+
+    let latestEpisodes = [];
+
+    for (let i = 0; i <= 5; i++) {
+        latestEpisodes.push(<Episode key={i} episode={episodes[i]} />);
+    }
+
     return (
         <>
-            <div className="latestEpContainer">
+            <div className="latestEpContainer glider">
```

Comment:
you can probably get rid of the `glider` class now that you aren't using that package anymore